lib/Doctrine/ORM/Tools/Console/Command/ClearCache/CollectionRegionCommand.php

```diff
@@ -24,13 +24,15 @@
  */
 class CollectionRegionCommand extends AbstractEntityManagerCommand
 {
+    /** @var string|null */
```

Comment:
Unfortunately I cannot put here `@var string`, because psalm fails

```

Error: lib/Doctrine/ORM/Tools/Console/Command/ClearCache/CollectionRegionCommand.php:28:22: NonInvariantDocblockPropertyType: Property Doctrine\ORM\Tools\Console\Command\ClearCache\CollectionRegionCommand::$defaultName has type string, not invariant with Symfony\Component\Console\Command\Command::$defaultName of type null|string (see https://psalm.dev/267)
```